test/integration/thruster.cc

```diff
@@ -147,6 +147,21 @@ void ThrusterTest::TestWorld(const std::string &_world,
   EXPECT_LT(sleep, maxSleep);
   EXPECT_TRUE(pub.HasConnections());
 
+  // input force cmd - this should be capped to 0
+  double forceCmd{-1000.0};
+  msgs::Double msg;
```

Comment:
This is causing a compile error

```
   /github/workspace/test/integration/thruster.cc: In member function 'void ThrusterTest::TestWorld(const string&, const string&, double, double, double, double, bool, double)':
  /github/workspace/test/integration/thruster.cc:172:16: error: redeclaration of 'ignition::msgs::Double msg'
     msgs::Double msg;
                  ^~~
```

Reply:
fixed. [d5b595b](https://github.com/ignitionrobotics/ign-gazebo/pull/1367/commits/d5b595b684935e56347c68f6734922a1675df03d)